src/main/java/com/launchdarkly/client/LoggerFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.launchdarkly.client;
+
+import java.util.concurrent.atomic.AtomicReference;
+import org.slf4j.Logger;
+
+public class LoggerFactory {
+
+    private static AtomicReference<ILoggerFactory> factory = new AtomicReference<>();
+
+    public static void install(ILoggerFactory factory) {
```

Comment:
Oh! I would much rather have this be part of a constructor somewhere. This way was about expediency to start the conversation. I can amend the PR on Monday. How long, do you think, before we'd see it in a release?